### PR TITLE
Enable support for 3scale/porta support for ppc64le(tested on UBI9)

### PR DESCRIPTION
**Pull request by shubham-bhagwat0**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by shubham-bhagwat0** [comment]
[porta-script-logs.txt](https://github.com/ppc64le/build-scripts/files/15148374/porta-script-logs.txt)

File: 3/3scale-porta/3scale-porta_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+        test/unit/payment_buyer_references_test.rb \
+        test/unit/three_scale/api/responder_test.rb \
+        test/unit/date_parsing_test.rb \
+        test/unit/services/audit_log_service_test.rb \
+        test/unit/cms/handler/markdown_test.rb \
+' TESTOPTS=--verbose --verbose --trace; then
+    echo "------------------$PACKAGE_NAME:install_success_but_unit_test_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Unit test Failure"
+    exit 1
```
Comment (AbhiIBM) [inline 3/3scale-porta/3scale-porta_ubi_9.3.sh:233]: Exit value should be 2 here.

File: 3/3scale-porta/3scale-porta_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+	  test/integration/master/api/services_controller_integration_test.rb \
+	  test/integration/provider/domains_controller_integration_test.rb \
+ 	  test/integration/provider/admin/dashboard/service/hits_controller_test.rb \
+ 	  test/integration/provider/passwords_integration_test.rb \
+        test/integration/by_provider_key_integration_test.rb \
+' TESTOPTS=--verbose --verbose --trace; then
+    echo "------------------$PACKAGE_NAME:install_success_but_integration_test_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Integration test Failure"
+    exit 1
```
Comment (AbhiIBM) [inline 3/3scale-porta/3scale-porta_ubi_9.3.sh:270]: Exit value should be 2 here.

File: 3/3scale-porta/build_info.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "Shubham.Bhagwat@ibm.com",
+    "package_name" : "3scale-porta",
+    "github_url": "https://github.com/3scale/porta",
+    "version": "3scale-2.14.1-GA",
+    "package_dir": "3scale/porta",
+    "default_branch": "main",
+    "build_script": "3scale-porta_ubi_9.3.sh",
+    "docker_build": "true",
```
Comment (AbhiIBM) [inline 3/3scale-porta/build_info.json:9]: Kept docker_build true but no Dockerfile is present in PR.

**Review by shubham-bhagwat0 — COMMENTED** [review]
changes made, fixed suggested issues

**Pull request closed without merging**